### DISCUSSION TITLE
Welcome to thread-tracker Discussions!

### DISCUSSION THREAD

**Discussion by vexx32** [vexx32/thread-tracker#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Wondercustard** [comment]
You're doing a great job! I wish I could have TiTi in all of my rp servers.

**Reply by vexx32** [reply]
If I can get some memory leaks (seemingly?) fixed, that will be Soon(tm).

**Reply by Wondercustard** [reply]
Oh no rush from me at all. Dev and UX is hard enough without catering to a larger entity. I look forward to seeing your creation grow!
